controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -261,11 +262,11 @@ func (r *NodeNetworkConfigurationPolicyReconciler) initializeEnactment(policy nm
 			return errors.Wrapf(err, "error waitting for NodeNetworkConfigurationEnactment: %+v", enactment)
 		}
 	} else {
-		enactmentConditions := enactmentconditions.New(r.Client, enactmentKey)
+		enactmentConditions := enactmentconditions.New(r.APIClient, enactmentKey)
 		enactmentConditions.Reset()
 	}
 
-	return enactmentstatus.Update(r.Client, enactmentKey, func(status *nmstateapi.NodeNetworkConfigurationEnactmentStatus) {
+	return enactmentstatus.Update(r.APIClient, enactmentKey, func(status *nmstateapi.NodeNetworkConfigurationEnactmentStatus) {
 		status.DesiredState = policy.Spec.DesiredState
 		status.PolicyGeneration = policy.Generation
 	})
```

Comment:
278/300 they should since we don't want to cache all the enactments at all the handlers, I am doing the change.
351 not needed since we are not going to have a mssive number of NNCPs on the cluster
361 NNSs are restricted by the cache selector so we have no problem there.